index.html

```diff
@@ -3,6 +3,44 @@
   <head>
     <meta charset="UTF-8">
     <title>Project</title>
+
+    <!------------------------------------------------------------------------------------------------------------------------------------------>
+    <script src="https://use.fontawesome.com/52e2986478.js"></script>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
I don't recommend to use jQuery. Delete it please.

Reply:
Done! 
But I'd like to know if there is a better way of creation slideToggle() effect in header.component.js and portfolio.component.js